src/__tests__/fixtures/color-case.compressed.css

```diff
@@ -1 +1 @@
-p{color:#AA93FE;background-color:#AFE}
\ No newline at end of file
+p{color:#AA93FE;background:#FFF url(/ImagesDir/loader.gif) center center no-repeat}
```

Comment:
During exploring the code, I noticed that we do not want to apply transforms to `compressed` format... or we do want it? :)

Anyways, it does not transform right now. Same for other transforms that I've committed.